bundles/net.tourbook/src/de/byteholder/geoclipse/mapprovider/DialogMP.java

```diff
@@ -143,11 +142,11 @@ protected String displayLogEntries(final ConcurrentLinkedQueue<LogEntry> logEntr
 
          sb.setLength(0);
          sb.append(TimeTools.getZonedDateTime(logEntry.time).format(_timeFormatter));
-         sb.append(COLUMN_SPACER);
+         sb.append(net.tourbook.common.UI.SPACE2);
          sb.append(logEntry.counter);
-         sb.append(COLUMN_SPACER);
+         sb.append(net.tourbook.common.UI.SPACE2);
          sb.append(logEntry.threadName);
-         sb.append(COLUMN_SPACER);
+         sb.append(net.tourbook.common.UI.SPACE2);
          sb.append(logEntry.tileEventId);

```

Comment:
When the fully qualified name must be used many times, then the code is easier readable when using a shortcut for this constant, e.g.

![mt-shorcut-for-constants](https://user-images.githubusercontent.com/1283445/91719719-206f2c80-eb96-11ea-8b0e-f334fb687319.png)

Reply:
Fixed